orange/src/test/java/com/hartwig/hmftools/orange/report/datamodel/VariantEntryFactoryTest.java

```diff
@@ -134,4 +134,56 @@ public void canDetermineTranscriptImpact()
                         .codingEffect(PurpleCodingEffect.MISSENSE)
                         .build()));
     }
+
+    @Test
+    public void shouldReportMultipleVariantsOnDriverGene()
```

Comment:
I would rename this function, `shouldGenerateMultipleVariantsForNonCanonicalAndCanonicalDriver`. 
The current name suggests this specific function relates to "reported", but that only holds for non-canonical drivers.  Also I would reflect that this function specifically checks canonical in combination with non-canonical drivers.